gif-screencast.el

```diff
@@ -121,13 +121,83 @@ various programs run here."
   :init-value nil
   :keymap gif-screencast-mode-map)
 
+(defvar gif-screencast--done nil
+  "Flag if capturing already done.")
+
+(defvar gif-screencast--finishing nil
+  "Flag if compilation in progress.")
+
+(defvar gif-screencast--counter 0
```

Comment:
This variable should be reset to 0 in `gif-screencast` in case the capture was interrupted.

Reply:
Ok. But we need to add new check for prevent new start when already running.

Reply:
Indeed.